admin/installer/setup_client.sh

```diff
@@ -15,12 +15,6 @@ sudo apt-get install -y postgresql-client
 curl -L https://github.com/docker/compose/releases/download/1.5.2/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
 chmod +x /usr/local/bin/docker-compose
 
-mkdir -p /home/ubuntu/postgres
```

Comment:
Chatted with Richard offline. Will wait for follow up to remove those depending on demo instructions.